### PR TITLE
Add Vasudev Jaiswal 

### LINKED ISSUE #641: Add VasudevJaiswal
<h3>

{ "name": "Vasudev Jaiswal",

    "githubUsername": "VasudevJaiswal",

    "imageUrl": "https://user-images.githubusercontent.com/85981735/129255772-5bac726e-d98e-4eb4-8c9f-c2206b72c976.png",

    "issueNumber": 630 }
</h3>

***

<h3> Here My github Profile Screenshot </h3>

***
VasudevJaiswal

### PR DESCRIPTION
**Pull request by VasudevJaiswal**
Add my profile data
Resolve #641

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1253,4 +1253,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/53567221/129094169-e012e3fb-3ba9-47e7-9173-0e09e7517034.png",
     "issueNumber": 642
   }
+  {
+    "name": "Vasudev Jaiswal",
+    "githubUsername": "VasudevJaiswal",
+    "imageUrl": "https://user-images.githubusercontent.com/53567221/129094169-e012e3fb-3ba9-47e7-9173-0e09e7517034.png",
```
Comment (Panquesito7) [inline data.json:1259]: Wrong URL image.
```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/85981735/129255772-5bac726e-d98e-4eb4-8c9f-c2206b72c976.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by schmelto — CHANGES_REQUESTED** [review]
Please see @Panquesito7 comment [here](https://github.com/EddieHubCommunity/awesome-github-profiles/pull/644#discussion_r688045572 )

**Comment by schmelto** [comment]
I've linked the issue #641 in the pr description thus it will automatically gets closed on pr merge.

**Comment by VasudevJaiswal** [comment]
Problem Resolved 😊 
Thanks @Panquesito7

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your contribution! 😄👍🎉
@schmelto, this PR can be merged. 👍

File: data.json [hunk 2]
```diff
@@ -1253,4 +1253,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/53567221/129094169-e012e3fb-3ba9-47e7-9173-0e09e7517034.png",
     "issueNumber": 642
   }
```
Comment (Panquesito7) [inline data.json:1255]: Oh, you forgot a comma here.
```suggestion
  },
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
LGTM. Thank you for your contribution! 😄👍🎉
@schmelto, this PR can be merged. 👍

**Review by schmelto — APPROVED** [review]
Lgtm

**Pull request merged**